RecoTracker/MkFit/plugins/HitSelectionWindows2017.h

```diff
@@ -0,0 +1,5868 @@
+#ifndef HitSelectionWindows_h
+#define HitSelectionWindows_h
+
+namespace mkfit {
+
+  struct HitSelectionWindows2017 {
```

Comment:
how specific to 2017 is this?

Reply:
Should be more of "Phase1" tracker than "2017" per se, will rename.